### ISSUE TITLE
Contrast text on form

### ISSUE THREAD

**Issue by AlexandraOM** [fac18/hussh#42]
The accessibility for the text on the whole webpage is overall really good, however when you get to the form, it is black text on a grey/black background and is therefore hard to read.

**Comment by marialani** [comment]
Font colour has been changed to white.
The font-weight has also been increased.
